xero-payroll-au.yaml

```diff
@@ -4979,14 +5003,47 @@ components:
       enum:
       - REGULATED  
       - SMSF
+    EmploymentType: 
+      type: string
+      enum:
+      - EMPLOYEE  
+      - CONTRACTOR
+    IncomeType: 
+      type: string
+      enum:
+      - SALARYANDWAGES  
+      - WORKINGHOLIDAYMAKER
+      - NONEMPLOYEE  
+      - CLOSELYHELDPAYEES
+      - LABOURHIRE  
+    TaxScaleType: 
+      type: string
+      enum:
+      - REGULAR  
+      - ACTORSARTISTSENTERTAINERS
+      - HORTICULTURISTORSHEARER  
+      - SENIORORPENSIONER
+      - WORKINGHOLIDAYMAKER  
+      - FOREIGN
+    WorkCondition: 
+      type: string
+      enum:
+      - PROMOTIONAL  
+      - THREELESSPERFORMANCESPERWEEK
+    SeniorMaritalStatus: 
+      type: string
+      enum:
+      - MEMBEROFCOUPLE  
+      - MEMBEROFILLNESSSEPARATEDCOUPLE
+      - SINGLE
     EmploymentBasis: 
       type: string
       enum:
       - FULLTIME  
       - PARTTIME  
       - CASUAL  
       - LABOURHIRE  
-      - SUPERINCOMESTREAM
```

Comment:
It sounds like SUPERINCOMESTREAM is still supported if the organisation hasn't yet enabled STP Phase 2. Should this be included until all AU orgs have converted to STP Phase 2? Otherwise we will have marshalling errors.

Reply:
Good catch. Part of the thinking was that partners would only bump to the major version with these changes when they were 100% ready for STP 2, but we weren't thinking about orgs taking a while to upgrade. We'll have to keep `SUPERINCOMESTREAM` and `WORKINGHOLIDAYMAKER` for now and make a subsequent PR in ~2023 when these values are no longer coming back down through the API.

Correct me if I'm wrong, but I don't think there's support for marking enum values as deprecated in an OpenAPI spec? So maybe we just put a comment on these two to flag them as deprecated for human readers.

Reply:
Resolved by adding WORKINGHOLIDAYMAKER, SUPERINCOMESTREAM back with a comment ✔️